db/structure.sql

```diff
@@ -2213,6 +2214,13 @@ CREATE INDEX index_projects_on_slug ON projects USING btree (slug);
 
 
 --
+-- Name: index_projects_on_state; Type: INDEX; Schema: public; Owner: -; Tablespace: 
+--
+
+CREATE INDEX index_projects_on_state ON projects USING btree (state);
```

Comment:
Is there a way to force it? db:migrate doesn't run anything in any environment and Travis seems to like the test db.